sys/Makefile.dep

```diff
@@ -105,9 +113,13 @@ ifneq (,$(filter fuzzing,$(USEMODULE)))
   USEMODULE += gnrc_pktbuf_malloc
 endif
 
-# include GNRC dependencies
-ifneq (,$(filter gnrc% %gnrc,$(USEMODULE)))
-  include $(RIOTBASE)/sys/net/gnrc/Makefile.dep
+ifneq (,$(filter gnrc_lorawan,$(USEMODULE)))
+  USEMODULE += xtimer
+  USEMODULE += random
+  USEMODULE += hashes
+  USEMODULE += crypto_aes_128
+  USEMODULE += netdev_layer
+  USEMODULE += gnrc_nettype_lorawan
```

Comment:
I think this is why the tests are failing now, all the dependency resolution of GNRC was removed, perhaps introduced while rebasing.

Reply:
Due to #16268 being merged. All GNRC-specify dependencies have been moved to their own file.

Reply:
Thanks ! Fixed.